base/applications/dxdiag/ddtest.c

```diff
@@ -366,7 +366,7 @@ VOID DDUpdateFrame(LPDIRECTDRAWSURFACE lpDDPrimarySurface ,LPDIRECTDRAWSURFACE l
     }
 }
 
-static LRESULT CALLBACK WindowProc(HWND hWnd, UINT msg, WPARAM wParam, LPARAM lParam)
+static LRESULT CALLBACK DDTestsWindowProc(HWND hWnd, UINT msg, WPARAM wParam, LPARAM lParam)
```

Comment:
Why is this wrapper here in the first place?
Can just pass `DefWindowProc`?
Other than that, file lgtm.

Reply:
Done.